pkg/ingester/metrics.go

```diff
@@ -505,6 +508,15 @@ func newTSDBMetrics(r prometheus.Registerer) *tsdbMetrics {
 			"Total number of out of order exemplar ingestion failed attempts.",
 			nil, nil),
 
+		tsdbOOOAppendedSamples: prometheus.NewDesc(
+			"cortex_ingester_tsdb_out_of_order_samples_appended_total",
+			"Total number of out of order samples appended.",
+			nil, nil),
+		tsdbTooOldSamples: prometheus.NewDesc(
+			"cortex_ingester_tsdb_too_old_samples_total",
+			"Total number of too old samples.",
```

Comment:
suggestion: Total number of samples rejected by TSDB append because they are too old.

question: how is it different from `validation.DiscardedSamples.WithLabelValues(sampleTooOld, userID)`?

Reply:
There can be some samples that are discarded in Commit() that would only be caught by the TSDB.

Reply:
If we do not care about that edge case much, we can remove this one

Reply:
Looks like we are not tracking the old out of order metrics from the TSDB. So to be consistent, I will remove this one from here.